apps/mobile/src/features/Send/common/useSendAddress.tsx

```diff
@@ -19,12 +19,7 @@ export const useSendAddress = () => {
     isLoading: isValidatingAddress,
     error: addressError,
     refetch,
-  } = useValidateAddress(
-    {address, chainId},
-    {
-      enabled: false,
-    },
-  )
+  } = useValidateAddress({address, chainId})
```

Comment:
this is no longer async, maybe can become that a regular function doesnt need to be a hoook anymore.